libs/common/Networking/GarnetTcpNetworkSender.cs

```diff
@@ -270,6 +270,8 @@ private unsafe void Send(Socket socket, GarnetSaeaBuffer sendObject, int offset,
             if (cnt > ThrottleMax)
                 throttle.Wait();
 
+            // Debug.WriteLine($"SEND: [{size}][{System.Text.Encoding.UTF8.GetString(new Span<byte>(sendObject.socketEventAsyncArgs.Buffer, offset, size)).Replace("\n", "|").Replace("\r", "")}]");
```

Comment:
Is this necessary?

Reply:
added so that protocol debugging is easier in future.